index.html

```diff
@@ -864,9 +864,12 @@
 </aside>
 
 A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define them. Their identity
-*in a [=context=]* is the set of characteristics they present to that context. People
-frequently present different identities to different contexts, and also frequently share an identity
-among several contexts.
+*in a [=context=]* is the set of characteristics they present to that context,
+directly or indirectly. People frequently present different identities to
+different contexts, and also frequently share an identity among several
+contexts. In certain cases, people may also choose to present minimal unique
+characteristics or false characteristics about themselves in certain
+[=contexts=] to achieve greater privacy on the web.
```

Comment:
How about this rewording? It groups the descriptions of the characteristics together, instead of putting the description of identity sharing and non-sharing in the middle.

```suggestion
*in a [=context=]* is the set of characteristics they present to that context,
directly or indirectly. This set of characteristics can be intentionally small
and can differ from the person's physical-world characteristics. People
frequently present different identities to
different contexts, and also frequently share an identity among several
contexts.
```

Reply:
I think it does a good job of highlighting some of the key points, but it does leave out that from a web pages perspective the identity of the user is actually the combination of the user's attributes and the UA's attributes. If we focus the concept of an identity too much on the user it seems like we lose the value of calling out that the UA is actually what gets used to uniquely identify users most of the time. I'd like to make that more understood in this text so that we're leveraging these principles to inform ourselves on better design choices in the future.

For example, if I configure my UA to behave as the example describes in [RFC 9110](https://www.rfc-editor.org/rfc/rfc9110.html#section-12.5.4) then that's describing a preference of the UA not me as I can understand `en-gb` the exact same as `en`, but this attribute could be significantly unique enough to fingerprint the UA, and by extension me the user if I'm the only user who uses this browser.

A way to modify your suggestion to align it better with this point would be the following:

> This set of characteristics can be intentionally small and can differ from the person's physical-world characteristics or [=identifiers=]. People frequently present different identities to different contexts, and also frequently share an identity among several contexts.

Does that align with where you wanted to take this @jyasskin

Reply:
I think you've just added "or [=identifiers=]" to my suggestion (I think in order to pull in its mention of "browser fingerprints"), and that sounds good to me.

Reply:
Cool, updating it now and then this one should be good to go.